FluentTerminal.App/Views/SettingsPages/About.xaml

```diff
@@ -52,36 +50,40 @@
                         Source="/Assets/AppIcons/Square106x106Logo.black.png"
                         Stretch="None" />
 
-                    <!--  Current version  -->
-                    <TextBlock Margin="0,24,0,8" Text="Current version" />
-                    <TextBlock><Run Text="{x:Bind ViewModel.CurrentVersion}" />
-                         - <Hyperlink NavigateUri="{x:Bind ViewModel.CurrentVersionReleaseNotesURL}">Release notes</Hyperlink></TextBlock>
+                    <StackPanel Margin="0,24,0,0">
+                        <TextBlock Margin="0,0,0,8" Text="Current version" />
+                        <TextBlock>
+                            <Run Text="{x:Bind ViewModel.CurrentVersion}" /> -
+                            <Hyperlink NavigateUri="{x:Bind ViewModel.CurrentVersionReleaseNotesURL}">Release notes</Hyperlink>
+                        </TextBlock>
+                    </StackPanel>
 
-                    <!--  Latest version  -->
-                    <TextBlock Margin="0,24,0,8" Text="Latest version" />
-                    <TextBlock Visibility="{x:Bind ViewModel.LatestVersionLoading, Mode=OneWay, Converter={StaticResource TrueToVisibleConverter}}"><Italic>loading...</Italic></TextBlock>
-                    <TextBlock Visibility="{x:Bind ViewModel.LatestVersionNotFound, Mode=OneWay, Converter={StaticResource TrueToVisibleConverter}}"><Italic>Not found</Italic></TextBlock>
-                    <TextBlock Visibility="{x:Bind ViewModel.LatestVersionFound, Mode=OneWay, Converter={StaticResource TrueToVisibleConverter}}">
-                        <Run Text="{x:Bind ViewModel.LatestVersion, Mode=OneWay}" />
-                         - <Hyperlink NavigateUri="{x:Bind ViewModel.LatestVersionReleaseNotesURL, Mode=OneWay}">
-                            Release notes
-                        </Hyperlink>
-                    </TextBlock>
+                    <StackPanel Margin="0,24,0,0">
+                        <TextBlock Margin="0,0,0,8" Text="Latest version" />
+                        <TextBlock Visibility="{x:Bind ViewModel.LatestVersionLoading, Mode=OneWay, Converter={StaticResource TrueToVisibleConverter}}">
+                            <Italic>loading...</Italic>
```

Comment:
Should this be capitalized? I feel like maybe "Checking for update..." would work better? Or maybe a spinner? I dunno. This just doesn't seem quite right.

Reply:
That's a good a suggestion, I'll definitely update the text and I'll see about a spinner